velox/exec/TableScan.cpp

```diff
@@ -16,6 +16,7 @@
 #include "velox/exec/TableScan.h"
 #include "velox/common/testutil/TestValue.h"
 #include "velox/common/time/Timer.h"
+#include "velox/connectors/hive/HiveConnectorSplit.h"
```

Comment:
There is a circular dependency problem with this change: `hive-connector -> exec -> hive-connector`, let me see if we can break out part of exec that is only required in `hive-connector`

Reply:
thanks @Yuhta updated

Reply:
hello @Yuhta is ok to merge the PR ?